reflection/Katas/Katas/Katas.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Linq;
+
+namespace Katas
+{
+    public class Katas
+    {
+        //The FizzBuzz Kata
+        public string FizzBuzz(int number) =>
+            number % 15 == 0 ? "FizzBuzz" :
+            number % 3 == 0 ? "Fizz" :
+            number % 5 == 0 ? "Buzz" :
+            number.ToString();
+
+        //The OddEven Kata
+        public string CheckNumber(int num)
+        {
+            if (num < 2)
+            {
+                return num.ToString();
+            }
+
+            bool isPrime = true;
+            for (int i = 2; i <= Math.Sqrt(num); i++)
+            {
+                if (num % i == 0)
+                {
+                    isPrime = false;
+                    break;
+                }
+            }
+
+            if (isPrime)
+            {
+                return num.ToString();
+            }
+            else if (num % 2 == 0)
+            {
+                return "Even";
+            }
+            else
+            {
+                return "Odd";
+            }        
+        }
+        
+        //The Leap Year Kata
+        public bool IsLeapYear(int year)
+        {
+            if (year % 4 != 0)
+            {
+                return false;
+            }
+            else if (year % 100 != 0)
+            {
+                return true;
+            }
+            else if (year % 400 == 0)
+            {
+                return true;
+            }
+            else
+            {
+                return false;
```

Comment:
in general, it's a good practice to have a single "exit"(return) in your methods, this simplifies the code. Try to change this and previous methods to have only one return statement, you can provide a local variable, that will store the value you want to return and set it depending on condition.